flake.nix

```diff
@@ -66,6 +72,42 @@
           "semver.nix"
           "LICENSE"
         ];
+        crane = [
+          "lib/buildDepsOnly.nix"
+          "lib/cargoBuild.nix"
+          "lib/mkCargoDerivation.nix"
+          "lib/mkDummySrc.nix"
+          "lib/writeTOML.nix"
+          "lib/cleanCargoToml.nix"
+          "pkgs/configureCargoCommonVarsHook.sh"
+          "pkgs/configureCargoVendoredDepsHook.sh"
+          "pkgs/inheritCargoArtifactsHook.sh"
+          "pkgs/installCargoArtifactsHook.sh"
+          "pkgs/remapSourcePathPrefixHook.sh"
+          "LICENSE"
+        ];
+        nix-std = [
```

Comment:
I personally don't like that crane requires this entire repo just for a `toTOML` function (afaik), maybe we could vendor our own `toTOML`...?

Reply:
I just tried in a repl and if I delete all files from nix-std except `{default,flake,list,serde,set,string}.nix`, the toTOML function still works.

So maybe we can just keep a limited number of files.

But yeah, obviously it would be better if we just had less dependencies.